Pod/UI/Classes/UILabel+BonMotUtilities.m

```diff
@@ -0,0 +1,37 @@
+//
+//  UILabel+BonMotUtilities.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/2/16.
+//
+//
+
+#import "BonMot.h"
```

Comment:
Since this is part of the BonMot library I think it should remain `#import "BonMot.h"`. The `<...>` syntax is intended for imports of files not provided by the project. See [this](http://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename) SO thread for more info.